master/getting-started/openstack/upgrade/downgrade.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Downgrading Calico
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/openstack/upgrade/downgrade'
+---
+
+## About downgrading {{site.prodname}}
+
+Under some circumstances, you may need to perform a downgrade and return your
+cluster to the previous version of {{site.prodname}}. If you need to downgrade
+you should do so as soon as possible to avoid an outage.
+
+> **Important**: After downgrading or aborting the migration it is necessary
+> to delete the previously migrated
+> [etcd](delete#deleting-calico-data-from-etcdv2-after-a-successful-migration-and-upgrade)
+{: .alert .alert-info}
+
+## calico-upgrade tool
+
+1. If you used the calico-upgrade tool, use the following command to abort the upgrade.
```

Comment:
This section is a little confusing. It sticks out a bit. Did you know that after running the calico-upgrade that it will actually prompt you to run this command? And we do mention that there on that page. So....are you sure we need to mention this here? If so, we might want to think about how to better integrate this section with those that follow, it's a bit of an odd duck here. I would also suggest that we think about how we're going to talk about the migration in the context of OpenStack, standardize on some pithy phrasing, and be consistent throughout all the sections in using that same phrase. (Also relates to intro page.)

Reply:
Its mentioned here like this because there's no requirement for them to have used `calico-upgrade` *at all* in an Openstack upgrade.